docs/reference/schema.md

```diff
@@ -514,6 +514,11 @@ The port of the binding.
 
 The protocol (`http` or `https`).
 
+#### `ip` (`string`)
```

Comment:
Would it be possible to add this same definition to the [`tye.yaml` YAML schema](https://github.com/dotnet/tye/blob/main/src/schema/tye-schema.json)?